### PR TITLE
Add functionality to all buttons

### PR DESCRIPTION
**Pull request by MiSwit13**
Every button will now perform the action on the label

### CODE REVIEW

**Comment by mabeechen** [comment]
congrats on your first PR

File: app/src/main/java/com/brazenhiker/richcalc/MainHostActivityFragment.java [hunk 1]
```diff
@@ -134,47 +137,59 @@ public void onClick(View view) {
...
+                    readOutText.setText(displayBuilder);
             }
         });
 
         plusButton = getView().findViewById(R.id.plus_button);
         plusButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                readOutText.setText("+");
+                    calc.addConstant(stringBuilderToDouble(displayBuilder));
```
Comment (mabeechen) [inline app/src/main/java/com/brazenhiker/richcalc/MainHostActivityFragment.java]: make a common method these three lines and reuse it for lines 162-164, 172-174, and 182-184.

**Comment by MiSwit13** [comment]
As part of this latest commit, I updated the onClickListeners to lambdas. The auto convert of the IDE appended numbers to the view variables. I wasn't sure if that mattered or if I should change them to be descriptive to the button being listened to. Let me know if you want them changed.

**Comment by mabeechen** [comment]
everything looks good to me.  Lets get it merged in.

**Review by mabeechen — APPROVED** [review]
good work!

**Pull request merged**